pyproject.toml

```diff
@@ -49,6 +49,7 @@ build-backend = "poetry.core.masonry.api"
 [tool.taskipy.tasks]
 start = "python manage.py run --debug"
 makemigrations = "python manage.py makemigrations"
+dockermigrations = "docker-compose exec web python /app/manage.py makemigrations"
```

Comment:
This is okay, but it would be nice if it started the service itself if it isn't already running.

Reply:
I agree with Mark here, took me a moment to figure out why the command wasn't doing anything 🙂

Reply:
Yea, that makes sense. We have two options here `docker-compose up` or `docker-compose up --no-recreate` or `docker-compose up --force-recreate`.

The former checks if there are any changes to pick up, if there are it recreates the containers then attaches to them. The latter attaches to the currently running containers and doesn't do any changes.

I'm leaning more towards the latter.

IE for the latter this task would become `docker-compose up web -d --no-recreate && docker-compose exec web python /app/manage.py makemigrations`

Reply:
You could just use `docker-compose run --rm makemigrations`.

Reply:
@ChrisLovering @MarkKoz was there ever a resolution to this topic?

Reply:
My suggested command should resolve Chris's concerns. I have used this command and it works well. It should also work if there's a container already running, since it will just start its own and remove it when its done.